docs/tornjak-helm-chart.md

```diff
@@ -0,0 +1,139 @@
+# Deploying Tornjak via SPIFFE Helm Charts Hardened
+
+## Overview
+
+You can deploy **Tornjak** using the [SPIFFE Helm Charts Hardened](https://github.com/spiffe/helm-charts-hardened)
+repository. 
+
+This guide walks you through deploying both the frontend and backend of Tornjak with Direct Access, 
+using Helm charts in a local Kubernetes environment via Minikube. By the end, you’ll have a working instance 
+of SPIRE integrated with Tornjak for easier visibility and management of your SPIFFE identities. 
+
+## Prerequisites
+
+Make sure you have the following installed on your system:
+
+- [Minikube](https://minikube.sigs.k8s.io/docs/start/?arch=%2Fmacos%2Fx86-64%2Fstable%2Fbinary+download)
+- [Helm](https://helm.sh/docs/intro/install/)
+
+---
+
+## Step-by-Step Deployment
+
+### 1. Start Minikube
+
+```bash
+minikube start
+``` 
+
+Once minikube is up you can continue to the next step.
+
+---
+
+### 2. Clone the Helm Charts Repository 
+
+Clone the `helm-charts-hardened` repository if you haven't already, then navigate 
+into it and create a new branch. 
+
+```bash
+git clone https://github.com/spiffe/helm-charts-hardened.git
+cd helm-charts-hardened 
+git checkout -b your-branch
+```
+
+You can name the branch whatever you like.
+
+---
+
+### 3. Create a Custom Values File
+
+In the root of your `helm-charts-hardened` directory, create a YAML file (e.g `your-values.yaml`).
+This file will define the Helm configurations Helm uses when deploying SPIRE. 
+
+Here's a template you can start with: 
```

Comment:
For this tutorial, we can be more prescriptive. So I would move this doc to a folder, and add a yaml file that is the same as what you used locally  

We want the quickstart to be as quick as possible! So removing the need for a person to write their own yaml will be quicker